integrations/Braze/browser.js

```diff
@@ -195,15 +190,15 @@ class Braze {
     });
 
     Object.keys(traits).forEach((key) => {
-      window.appboy.getUser().setCustomUserAttribute(key, traits[key]);
+      window.braze.getUser().setCustomUserAttribute(key, traits[key]);
```

Comment:
For array they are using a different function [setCustomAttributeArray](https://www.braze.com/docs/developer_guide/platform_integration_guides/web/analytics/setting_custom_attributes/#implementation-examples-1:~:text=Setting%20a%20custom%20attribute%20with%20an%20array%20value).